go.mod

```diff
@@ -11,12 +11,14 @@ require (
 	github.com/cenkalti/backoff v1.1.1-0.20190506075156-2146c9339422 // indirect
 	github.com/containerd/cgroups v0.0.0-20201119153540-4cbc285b3327 // indirect
 	github.com/containerd/console v1.0.1 // indirect
-	github.com/containerd/containerd v1.3.9 // indirect
+	github.com/containerd/containerd v1.4.3 // indirect
```

Comment:
Bumping up containerd to 1.4 breaks compatibility with 1.3 which we still need to support. Is there anything you need from 1.4?